core/src/main/scala/zio/config/string.scala

```diff
@@ -0,0 +1,114 @@
+package zio.config
+
+import java.io.File
+import java.net.{ URI, URL }
+import java.time.{ Instant, LocalDate, LocalDateTime, LocalTime }
+import java.util.UUID
+
+import scala.concurrent.duration.Duration
+
+object string extends ReadFunctions with ConfigDocsFunctions {
+  type K = String
+  type V = String
+
+  val bigDecimal: ConfigDescriptor[BigDecimal] =
+    Source(ConfigSource.empty, PropertyType.BigDecimalType) ?? "value of type bigdecimal"
```

Comment:
`PropertyType` is `V` type based and it's sill on top level without module.

Reply:
PropertyType. BigDecimalType can convert the value which can only a string.
I think the instances of `propertytype` should be brought back to `string` (or `default`) module